glide.yaml

```diff
@@ -10,7 +10,7 @@ excludeDirs:
 - toolbox
 import:
 - package: github.com/coreos/etcd
-  version: release-3.1
+  version: 20490caaf0dcd96bb4a95e40625559def8ef5b04
```

Comment:
This is to avoid a collision between this and Kubernetes' etcd. The reason is that we are also pulling in kubernetes and this version of kubernetes uses this hash for etcd. https://github.com/kubernetes/kubernetes/blob/master/Godeps/Godeps.json#L452.
You are right about testing standalone. @dangula do we have a test for standalone?